README.md

```diff
@@ -1,6 +1,6 @@
 ## Template
```

Comment:
### [OPTIONAL]

Please replace the `Template` with your project name i.e., `Leaderboard` so that visitors will know the README is about the Leaderboard project. 👍🏻